plugins/catalog-import/api-report.md

```diff
@@ -116,11 +127,10 @@ export class CatalogImportClient implements CatalogImportApi {
   }>;
 }
 
-// Warning: (ae-forgotten-export) The symbol "StepperProviderOpts" needs to be exported by the entry point index.d.ts
 // Warning: (ae-missing-release-tag) "CatalogImportPage" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const CatalogImportPage: (opts: StepperProviderOpts) => JSX.Element;
+export const CatalogImportPage: () => JSX.Element;
```

Comment:
Here is a breaking change. While I could keep the parameters as is, it couldn't find I way to make it work. If I just keep the parameter to keep compatibility, but not keep the logic it would still be a breaking change. So I think going with a breaking change is a better choice.